.github/workflows/csharp-exercise-report.yml

```
@@ -11,6 +11,7 @@ on:
 
 jobs:
   update-csharp-exercises:
+    name: Update csharp exercise report
```

```suggestion
    name: Update exercise report
```

If it's scoped to C#, do we need the extra "csharp" here?

---

Nope, we don't, I just copied the old name. Good suggestion :)